force-app/main/dialogue/internal-components/lwc/hot_messagingMessageComponent/hot_messagingMessageComponent.js

```diff
@@ -31,8 +32,19 @@ export default class CrmMessagingMessageComponent extends LightningElement {
             this.threads = result.data;
         }
     }
+
+    accountId;
+    accountResult;
+    @wire(getAccountOnRequest, { recordId: '$recordId' })
```

Comment:
På sikt kan det hende at denne komponenten skal ligge på ServiceAppointment eller WorkOrder. Vi må sørge for at denne metoden ikke tryner om den ligger på et annet objekt.